include/utility/json_utility.hpp

```diff
@@ -638,5 +638,59 @@ inline bool isBackupAndRestoreRequired(const nlohmann::json& i_sysCfgJsonObj)
     }
     return false;
 }
+
+/**
+ * @brief API to get PCA chip command with details taken from JSON.
+ *
+ * This API can be used to get PCA chip bind/unbind command for the given FRU
+ * path with the help of PCAChipAddress, busType and driverType tags taken from
+ * system config JSON.
+ *
+ * @param[in] i_fruPath - FRU path.
+ * @param[in] i_sysCfgJsonObj - System config JSON object.
+ * @param[in] i_action - Action parameter which takes "bind/unbind" as its
+ * value.
+ *
+ * @return On success returns PCA chip bind/unbind command. On failure returns
+ * empty string.
+ */
+inline std::string getPCAChipCmd(const std::string& i_fruPath,
```

Comment:
Return type can possibly be `std::optional<std::string>` . Caller can check for `has_value()` to check error condition. Just a recommendation for better readability.

Reply:
done